src/main/kotlin/org/kamiblue/client/module/modules/combat/CrystalAura.kt

```diff
@@ -240,8 +246,13 @@ internal object CrystalAura : Module(
             }
 
             if (it.phase == TickEvent.Phase.END) {
-                if (inactiveTicks > 5 || getHand() == EnumHand.OFF_HAND) PlayerPacketManager.resetHotbar()
-                if (inactiveTicks > 20) resetRotation()
+                if (getHand() == EnumHand.OFF_HAND) {
+                    resetHotbar()
+                }
+                if (inactiveTicks > 20) {
+                    resetHotbar()
+                    resetRotation()
+                }
```

Comment:
Are you supposed to be resetting hotbar twice?

Reply:
HotbarManager will handle that itself